Appendix review

Fietel opened the pull request:
Gonna review the rest of the chapter over the course of the next hours/day. Just wanted to start a small discussion:
The screenshot of Figure 14.12 is imo not really good. The text mentions that ... _[t]he account can only open one "p2.xlarge" instance per region_ but the screenshot shows a limit of 52 instances.
Am I correct in assuming that it is due to already increased limits? 
Since I didnt request an upgrade, I could update the screenshot with the default setting to make it consistent with the text.

astonzhang commented:
Just leave the Figures in this section. We'll probably replace some of them in the future.

Fietel commented:
Ah okay, thought it was ready for review after the latest commits of @smolix :)
PR is ready:
1. Changed some phrasing. Felt like the steps were introduced quite randomly, not 100% happy with my version either, but I would definitely recommend adding such information about it.
2. Changed occurences of `apt-get` to `apt` (consistency) and added the `-r` flag to the removal of cuda (didn't test that one but I assume that it might be difficult without ;D)

Apart from that, the section looks fine to me. Happy Easter!

Fietel commented:
Completed the rest of the appendix. Mainly consistency/punctuation, some minor text improvements and replacement of outdated content. `math.md`missed a link, I have added https://github.com/juanklopper/MIT_OCW_Linear_Algebra_18_06, assuming that this is where it should link (Commit 8f42baf contains nothing else and can be reverted if I am wrong ;))

Would continue with either Chap1,2,3 or Chap 6. Which one would you prefer?

smolix commented:
> Gonna review the rest of the chapter over the course of the next hours/day. Just wanted to start a small discussion:
> The screenshot of Figure 14.12 is imo not really good. The text mentions that ... _[t]he account can only open one "p2.xlarge" instance per region_ but the screenshot shows a limit of 52 instances.
> Am I correct in assuming that it is due to already increased limits?
> Since I didnt request an upgrade, I could update the screenshot with the default setting to make it consistent with the text.

Touche ... that's my own account and in it, yes, I have a quota of 52 machines. I didn't feel like photoshopping it but I obviously could. I was hoping that the readers were going to be able to deal with it but if it's trouble, I can redo this in a region where I don't have quota :-)

smolix commented:
> Completed the rest of the appendix. Mainly consistency/punctuation, some minor text improvements and replacement of outdated content. `math.md`missed a link, I have added https://github.com/juanklopper/MIT_OCW_Linear_Algebra_18_06, assuming that this is where it should link (Commit [8f42baf](https://github.com/d2l-ai/d2l-en/commit/8f42baf8cfe8c2a36cd50b69c04e75312216e665) contains nothing else and can be reverted if I am wrong ;))
> 
> Would continue with either Chap1,2,3 or Chap 6. Which one would you prefer?

Go over it sequentially. That's probably the best. We're still improving / adding / editing the later chapters quite a bit. And awesome job!

astonzhang commented on chapter_appendix/aws.md:
````diff
@@ -85,7 +85,7 @@ Your server is ready now.
 
 ```
-sudo rm /usr/local/cuda
+sudo rm -r /usr/local/cuda
````
no need -r

Fietel replied:
Interesting, thanks!

astonzhang commented on chapter_appendix/aws.md:
```diff
@@ -74,8 +74,8 @@ Here, "/path/to/key.pem" is the path of the locally-stored key used to access th
-sudo apt-get update
-sudo apt-get dist-upgrade
+sudo apt update
+sudo apt dist-upgrade
```
let's keep apt-get. for instance, "apt dist-upgrade" should be "apt full-upgrade"

Fietel replied:
Missed that one, true:) The two occurrences of `dist-upgrade` in `aws.md` should be the only mistakes tho.
There was already a mix of `apt` and `apt-get`, I just wanted to keep it consistent

Fietel commented:
> Touche ... that's my own account and in it, yes, I have a quota of 52 machines. I didn't feel like photoshopping it but I obviously could. I was hoping that the readers were going to be able to deal with it but if it's trouble, I can redo this in a region where I don't have quota :-)

Well since it is highlighted in the picture and mentioned in the text, I would definitely recommend changing it :D 
I've already made a screenshot of the limits earlier, might save you some work.
![ec2limit](https://user-images.githubusercontent.com/23087465/56608916-281ada80-660c-11e9-91a4-8dfc2b860cdc.png)
